Grid.hpp

```diff
@@ -67,18 +67,23 @@ class LIB_EXPORT Grid {
      *
      * @param comm MPI communicator.
      * @param filename Grid file in netCDF format.
-     * @param dim0_name Name of 1st grid dimension in netCDF file (optional)
-     * @param dim1_name Name of 2nd grid dimension in netCDF file (optional)
+     * @param xdim_name Name of 1st grid dimension in netCDF file (optional)
+     * @param ydim_name Name of 2nd grid dimension in netCDF file (optional)
+     * @param dim_order Permutation for ordering of dimensions.
      * @param mask_name Name of land mask variable in netCDF file (optional)
+     * @param ignore_mask Should the land mask be ignored?
+     * @param px Is the domain periodic in the x-direction?
+     * @param py Is the domain periodic in the y-direction?
```

Comment:
```suggestion
     * @param px Is the domain periodic in the x-direction.
     * @param py Is the domain periodic in the y-direction.
```

Just in case we confuse future selves :sweat_smile:

Reply:
Oops missed your suggestion here. Already fixed in https://github.com/nextsimhub/domain_decomp/pull/60/commits/e8bf9af79e15c0b815d33777f951b56aaccd43f1.